distribution/src/main/resources/drill-config.sh

```diff
@@ -213,6 +213,9 @@ fi
 # Set Drill-provided defaults here. Do not put Drill defaults
 # in the distribution or user environment config files.
 
+# Prefer IPv4 over IPv6
+export DRILL_JAVA_OPTS=${$DRILL_JAVA_OPTS:-" -Djava.net.preferIPv4Stack=true -Djava.net.preferIPv6Addresses=false"}
```

Comment:
Please use the `DRILLBIT_OPTS` variable below instead of `DRILL_JAVA_OPTS` and move it to the part where some other specific properties are set, like `ReservedCodeCacheSize` and so on.

Reply:
Isnt his important for both... drillbit and sqlline? 

And i thought this has to be done before including drill-env.sh ?

Reply:
ah, i see... there is also fine but this will make it impossible for the user to overwrite it in drill-env.sh